aredis/pubsub.py

```diff
@@ -249,7 +249,7 @@ def run_in_thread(self, daemon=False):
         for pattern, handler in iteritems(self.patterns):
             if handler is None:
                 raise PubSubError("Pattern: '%s' has no handler registered")
-        loop = asyncio.get_event_loop()
+        loop = self.connection.loop
```

Comment:
Since it is the problem of method how user uses the client, I think it is better to raise the error than to handle it in the client code. User will be reminded.
So just keep it intact~